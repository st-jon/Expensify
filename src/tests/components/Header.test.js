import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'
import Header from '../../components/Header'

test('Should render Header', () => {
    const wrapper = shallow(<Header />)
    expect (toJSON(wrapper)).toMatchSnapshot()

    // expect(wrapper.find('h1').text()).toBe('EXPENSIFY')
    // const renderer = new ReactShallowRenderer()
    // renderer.render(<Header />)
    // expect(renderer.getRenderOutput()).toMatchSnapshot()
})
