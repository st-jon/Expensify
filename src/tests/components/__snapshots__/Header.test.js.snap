// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Header 1`] = `
<header>
  <h1>
    EXPENSIFY
  </h1>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    exact={true}
    to="/"
  >
    Dashboard 
  </NavLink>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    to="/create"
  >
    Create Expense 
  </NavLink>
</header>
`;
